hydra-node/test/Hydra/NetworkSpec.hs

```diff
@@ -137,6 +137,62 @@ spec = do
               waitFor NetworkConnected
               waitFor $ PeerConnected carolConfig.advertise
 
+      it "tracks peer disconnects/connections correctly" $ \tracer -> do
```

Comment:
Why not expand the one test above? Seems very redundant to the `emits connectivity events`